src/battle_ai_main.c

```diff
@@ -2194,24 +2197,26 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
         case EFFECT_PROTECT:
             {
                 bool32 decreased = FALSE;
-                switch (move)
+                enum ProtectMethod protectMethod = GetMoveProtectMethod(move);
+                switch (protectMethod)
                 {
-                case MOVE_QUICK_GUARD:
-                case MOVE_WIDE_GUARD:
-                case MOVE_CRAFTY_SHIELD:
+                case PROTECT_QUICK_GUARD:
+                case PROTECT_WIDE_GUARD:
+                case PROTECT_CRAFTY_SHIELD:
                     if (!isDoubleBattle)
                     {
                         ADJUST_SCORE(-10);
                         decreased = TRUE;
                     }
```

Comment:
Fall Through? `MAT_BLOCK` would decrease the score twice here.

Reply:
> Fall Through? `MAT_BLOCK` would decrease the score twice here.

Corrected, should not be a fallthrough